pkg/apis/extensions/v1beta1/register.go

```diff
@@ -60,6 +58,7 @@ func addKnownTypes() {
 		&ListOptions{},
 		&ConfigMap{},
 		&ConfigMapList{},
+		&v1.DeleteOptions{},
```

Comment:
> Remember, delete options are query based and have no innate type meta.
> They exist as a schema tied to every version and group.

Without innate `TypeMeta` it seems like I'd express my preferred decoding target in the API server and write my code to that target.  I'm not seeing why I would choose to cross register.  I have a logical dependency, lets express that and expose it via swagger.